vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroup.java

```diff
@@ -71,6 +72,32 @@
 /**
  * Radio Button Group allows the user to select exactly one value from a list of
  * related but mutually exclusive options.
+ * <h2>Validation</h2>
+ * <p>
+ * Radio Button Group comes with a built-in validation mechanism that verifies
+ * that a radio button is selected when
+ * {@link #setRequiredIndicatorVisible(boolean) required} is enabled. Validation
+ * is triggered whenever the user selects a radio button or the value is updated
+ * programmatically. In practice, however, the required error can only occur if
+ * the value is cleared programmatically. This is because radio buttons, by
+ * design, don't allow users to clear a selection through UI interaction. If the
+ * required error occurs, the component is marked as invalid and an error
+ * message is displayed below the input.
```

Comment:
Same as for checkbox group, I'd not use "input" here.

```suggestion
 * message is displayed below the group.
```

Reply:
Fixed.